R/api_cache.R

```diff
@@ -0,0 +1,98 @@
+#' @title Data Caching API
+#' @description Functions to manage cache data across sessions.
+#' @param path character, path to cache directory, Default: cache_dir()
+#' @param \dots arguments passed to [list.files][base::list.files]
```

Comment:
```suggestion
#' @param \dots arguments passed to [list.files()]
```
(base does not need to be prefixed)